CactbotOverlay/CactbotOverlay.cs

```diff
@@ -247,6 +253,90 @@ public CactbotOverlay(CactbotOverlayConfig config)
       fast_update_timer_semaphore_.Release();
     }
 
+    public void EnableWatchers(bool check)
+    {
+      if (!init_ || this.watchers == null)
```

Comment:
I think if you initialize watchers in its declaration to be [], then this function could be much shorter and look like this:
```
public void EnableWatchers(bool check) {
  this.watcher_state = check;
  foreach (var watcher in this.watchers)
    watcher.EnableRaisingEvents = check;
}
```